core-domain/src/testFixtures/java/gohigher/application/ProcessFixture.java

```diff
@@ -13,38 +13,46 @@
 @RequiredArgsConstructor
 public enum ProcessFixture {
 
-	TO_APPLY(ProcessType.TO_APPLY, "", null),
-	DOCUMENT(ProcessType.DOCUMENT, "", LocalDateTime.now().plusDays(6)),
-	TEST(ProcessType.TEST, "", LocalDateTime.now().plusDays(10)),
-	INTERVIEW(ProcessType.INTERVIEW, "", LocalDateTime.now().plusDays(20)),
-	COMPLETE(ProcessType.COMPLETE, "", LocalDateTime.now().plusDays(40)),
+	TO_APPLY(ProcessType.TO_APPLY, "지원 예정", null, 0),
```

Comment:
현재 코드에서 order 0은 할당받이 않은 것을 의미하므로 fixture 또한 1로 시작해야 할듯합니다!

```suggestion
	TO_APPLY(ProcessType.TO_APPLY, "지원 예정", null, 1),
```

Reply:
이 부분을 놓쳤군요...
변경 완료했습니다!